### ISSUE TITLE
CLI results in 503 http status error

### ISSUE THREAD

**Issue by thepassle** [liamcottle/rustplus.js#67]
Hi, this likely isn't an issue with the library, but I'm trying to get setup with using rustplus.js. Following the documentation, to get my `playerToken`, I'm getting:

```
⚡ npx @liamcottle/rustplus.js fcm-register
Registering with FCM

node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^
<ref *5> StatusCodeError: 503 - "{\n  \"error\": {\n    \"code\": 503,\n    \"message\": \"The service is currently unavailable.\",\n    \"status\": \"UNAVAILABLE\"\n  }\n}\n"
    at new StatusCodeError (/Users/au87xu/rust-thing/node_modules/request-promise-core/lib/errors.js:32:15)
    at plumbing.callback (/Users/au87xu/rust-thing/node_modules/request-promise-core/lib/plumbing.js:104:33)
    at Request.RP$callback [as _callback] (/Users/au87xu/rust-thing/node_modules/request-promise-core/lib/plumbing.js:46:31)
    at self.callback (/Users/au87xu/rust-thing/node_modules/request/request.js:185:22)
    at Request.emit (node:events:517:28)
    at Request.<anonymous> (/Users/au87xu/rust-thing/node_modules/request/request.js:1154:10)
    at Request.emit (node:events:517:28)
    at IncomingMessage.<anonymous> (/Users/au87xu/rust-thing/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (node:events:631:28)
    at IncomingMessage.emit (node:events:529:35)
    at endReadableNT (node:internal/streams/readable:1400:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  statusCode: 503,
  error: '{\n' +
    '  "error": {\n' +
    '    "code": 503,\n' +
    '    "message": "The service is currently unavailable.",\n' +
    '    "status": "UNAVAILABLE"\n' +
    '  }\n' +
    '}\n',
  options: {
    url: 'https://fcm.googleapis.com/fcm/connect/subscribe',
    method: 'POST',
    headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
    form: {
      authorized_entity: '976529667804',
      endpoint: 'https://fcm.googleapis.com/fcm/send/dkhxxSCGxIU:APA91bGjbXy68u2hfytRc7Gxb2O-F7oZutqGYX-VQxKqRc-5ZPIfbUFpItiKCpOP3UAYhd2IE9pUm2m5wigv1bKBflSNO3193NRMH97J2Bk9RInyxz9eyv3CgoqKwhiMDKb1qzx-Vlsw',
      encryption_key: 'BMJfu8kpIGo6uLr-1Hcdb7AGrafkcGPkbkZ5OqBdQ3jgb44MpW3oQpZ_KXNji9Nxf4Y81cR_qMxwoK4GX9E5y_w',
      encryption_auth: 'j_vFFMsApnf8e4osYcN0rw'
    },
    callback: [Function: RP$callback],
    transform: undefined,
    simple: true,
    resolveWithFullResponse: false,
    transform2xxOnly: false
  },
```

The reason I'm creating this issue is because getting a (specifically) `503` error for this seem suspect, and I'm wondering if anybody has ran into this before.

**Comment by liamcottle** [comment]
This should now be fixed in v2.5.0

**Comment by thepassle** [comment]
`npx @liamcottle/rustplus.js@2.5.0 fcm-register` still only gives me a `503` error, its not really clear to me how to get the `playerToken`
